pkg/services/provisioning/notifiers/config_reader_test.go

```diff
@@ -31,6 +32,12 @@ func TestNotificationAsConfig(t *testing.T) {
 	Convey("Testing notification as configuration", t, func() {
 		sqlstore.InitTestDB(t)
 
+		for _, index := range []int{1, 2, 3, 4} {
```

Comment:
I think the following is more concise and less prone to mistakes:
```suggestion
		for i := 1; i < 5; i++ {
```